2.5/Dockerfile.fedora

```diff
@@ -8,7 +8,6 @@ EXPOSE 8080
 ENV NAME=ruby \
     RUBY_VERSION=2.5 \
     VERSION=0 \
-    RELEASE=1 \
```

Comment:
@omron93, is this the release you meant to be removed?

Reply:
Leaving this here as a reference:
https://fedoraproject.org/wiki/Container:Guidelines#VERSIONING